### PR TITLE
LPS-167973 Upgrade process to remove stale PWC versions

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Review by marcogalluzzi — APPROVED** [review]

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/upgrade/v3_2_6/test/DeleteStalePWCVersionsUpgradeProcessTest.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+		_checkOutFileEntry(fileEntry1);
+		_checkOutFileEntry(fileEntry2);
+		_checkOutFileEntry(fileEntry3);
+
+		Assert.assertTrue(_hasPWCVersion(fileEntry1));
+		Assert.assertTrue(_hasPWCVersion(fileEntry2));
+		Assert.assertTrue(_hasPWCVersion(fileEntry3));
+
+		_checkInFileEntry(fileEntry1);
```
[the remark this answers is not in the document]
Comment (adolfopa) [reply modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/upgrade/v3_2_6/test/DeleteStalePWCVersionsUpgradeProcessTest.java:87]: It is testing that active PWCs are not deleted. There's no way to test the other side of the coin with an integration test unless we add a fake PWC after checking in. I think it is worth it, so I'll add that change later.

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia @marcogalluzzi Updated the test to check that stale versions are actually removed. No other changes.

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:upgrade

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**